src/components/Errors.tsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+
+type Props = {
+  hasError: boolean;
+  setHasError: (value: boolean) => void;
+  errorMessage: string;
+};
+
+export const Errors:React.FC<Props> = ({
+  hasError,
+  setHasError,
+  errorMessage,
+}) => (
+  <div
+    data-cy="ErrorNotification"
+    className="notification is-danger is-light has-text-weight-normal"
+    hidden={!hasError}
```

Comment:
It's better to use render condition instead of hidden attribute. Just don't render this component at all if you don't have an error